### PR TITLE
Move custom editor tests back into VS Code

### PR DESCRIPTION
**Pull request by mjbvz**
Moves the tests from https://github.com/mjbvz/vscode-experimental-webview-editor-extension into the VS Code repo

### CODE REVIEW

File: extensions/vscode-api-tests/src/extension.ts [hunk 1]
```diff
@@ -29,6 +30,7 @@ export function activate(context: vscode.ExtensionContext) {
 
 		vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(`memfs:/sample-folder/large.ts`));
```
Comment (mjbvz) [inline extensions/vscode-api-tests/src/extension.ts]: @weinand @bpasero Can either of you explain why extension activation is disabled for node.js? Should my custom editor registration live somewhere else?

Comment (bpasero) [reply extensions/vscode-api-tests/src/extension.ts]: @mjbvz I do not fully understand that question but the file you are linking to is only used for VSCode for Web to have a demo setup, e.g. used for https://vscode-web-test-playground.azurewebsites.net/?enter=true

Comment (bpasero) [reply extensions/vscode-api-tests/src/extension.ts]: Please do not make changes to this file unless you want to support custom editors in web!

**Review by bpasero — CHANGES_REQUESTED** [review]

File: extensions/vscode-api-tests/src/extension.ts (hunk 1 above)
Comment (mjbvz) [reply extensions/vscode-api-tests/src/extension.ts]: To test custom editors end to end, I'd like to have a contribution point and a basic extension that registers a custom editor

I thought this this should live under `vscode-api-test` along with our other API tests (such as the webview tests). But it looks like that extension's activation is used for web and I don't understand why it is under `vscode-api-test`

Where should I to put my test custom editor extension instead? Do we have an other similar tests that create test extensions?

Comment (bpasero) [reply extensions/vscode-api-tests/src/extension.ts]: @mjbvz yes it is the right location. Your test should be inside `extensions/vscode-api-tests/src/singlefolder-tests` and the activation goes into `extensions/vscode-api-tests/package.json`.

We should probably rename `extensions/vscode-api-tests/src/extension.ts` to `extensions/vscode-api-tests/src/THIS_FILE_IS_ONLY_USED_IN_WEB_PLAYGROUND.ts`, it is unrelated to running tests!

Comment (bpasero) [reply extensions/vscode-api-tests/src/extension.ts]: I have moved `extension.ts` into a folder called `web-playground` to make it clearer.

**Comment by bpasero** [comment]
@mjbvz snow-plow alert:

image

**Comment by mjbvz** [comment]
Closing as out of date

**Pull request closed without merging**